assets/scss/templates/_guidelines.scss

```diff
@@ -159,4 +160,114 @@
     margin-bottom: 10px;
     text-align: left;
   }
+}
+
+.inputs-card {
+  width: 370px;  
+  h5 {
+    margin: 15px 0;
+    font-size: $dp-font-size-p;
+    font-family: $dp-font-family;
+    font-weight: bold;
+  }
+
+  p {
+    font-size: $dp-font-size-small;
+    line-height: 18px;
+    color:$dp-color-grey;
+  }
+}
+
+.guideline-focus{
+      color: $dp-color-grey;
+      border: 1px solid $dp-color-yellow !important;
+      border-radius: 3px;
+      -webkit-box-shadow: 0 0 0 2px rgba($dp-color-darkgrey,0.1);
+      box-shadow: 0 0 0 2px rgba($dp-color-darkgrey,0.1);
+      outline: none;
+}
+
+.guideline-error {
+  border: 1px solid $dp-color-red !important;
+}
+
+.guideline-error-txt {
+  font-family: $dp-font-family-proximanova;
+  color:$dp-color-red;
+  font-size: $dp-font-size-p;
+  line-height: 20px;
+}
+
+.guideline-check-label {
+  color:$dp-color-grey;
+  a {
+    color:$dp-color-lightgreen !important;
```

Comment:
Is this `!important` really necessary? we try to avoid this at all costs, could you please re check? The same with the others `!important`

Reply:
In this stylesheet I used `!important` to "force" the states of the form elements

Reply:
Why not add a class for this?

Reply:
we should not have! important in css, and if it is to show a specific state, I prefer to create a class for a correct visualization